internal/remotewrite/remote_write_url_test.go

```diff
@@ -58,8 +59,16 @@ func TestRemoteWriteURL(t *testing.T) {
 			Staging:        false,
 			FedRAMP:        false,
 			LicenseKey:     "non-eu-license-key",
-			Expected:       "https://metric-api.newrelic.com/prometheus/v1/write?prometheus_server=source",
 			DataSourceName: "source",
+			Expected:       "https://metric-api.newrelic.com/prometheus/v1/write?prometheus_server=source",
+		},
+		{
+			Name:          "collectorName",
+			Staging:       false,
+			FedRAMP:       false,
+			LicenseKey:    "non-eu-license-key",
+			CollectorName: "foo",
```

Comment:
is this a possible use-case? as far as I understand no user config could have this changed.